github/orgs_teams_test.go

```diff
@@ -508,7 +509,34 @@ func TestOrganizationsService_ListPendingTeamInvitations(t *testing.T) {
 		testMethod(t, r, "GET")
 		testFormValues(t, r, values{"page": "1"})
 		testHeader(t, r, "Accept", mediaTypeOrgMembershipPreview)
-		fmt.Fprint(w, `[{"id":1}]`)
+		fmt.Fprint(w, `[
```

Comment:
I personally don't mind having all the fields here, since they come directly from GitHub API examples.

If we reduce it to just IDs, keep the new inviter field with an ID too.